src/System.Net.Http/tests/FunctionalTests/CustomContent.cs

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Text;
+
+namespace System.Net.Http.Functional.Tests
+{
+    public class CustomContent : StreamContent
```

Comment:
In the case of a type with virtual methods it make things easier to read and analyze (no need to go look for other subclasses), not to mention the potential perf gains, so I consider it good practice. Up to you.